src/cdk/overlay/overlay-directives.ts

```diff
@@ -272,7 +272,12 @@ export class ConnectedOverlayDirective implements OnDestroy, OnChanges {
 
   /** Builds the overlay config based on the directive's inputs */
   private _buildConfig(): OverlayState {
-    let overlayConfig = new OverlayState();
+    const positionStrategy = this._position = this._createPositionStrategy();
+    const overlayConfig = new OverlayState({
```

Comment:
That's what I did. The `position` in this case is different, because we want to save a reference to it before we pass it to the `OverlayRef`.